SPECS/bazel/bazel.spec

```diff
@@ -49,6 +49,9 @@ cp ./scripts/packages/bazel.sh %{buildroot}/%{_bindir}/bazel
 %attr(0755,root,root) %{_bindir}/bazel-real
 
```

Comment:
I don't think these auto upgrades were intended to be here?

Reply:
I think they showed up due to the wrong target branch